utilities-dotnet/utilities-dotnet/Auth.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Cysharp.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace AMRC.FactoryPlus.ServiceClient;
+
+/// <summary>
+/// The body needed for to POST a principal
+/// </summary>
+public struct PostPrincipalBody
+{
+    /// <summary>
+    /// UUID of the principal
+    /// </summary>
+    public Guid Uuid;
+    /// <summary>
+    /// The Kerberos string
+    /// </summary>
+    public string Kerberos;
+    
+    public PostPrincipalBody(Guid uuid, string kerberos)
+    {
+        this.Uuid = uuid;
+        this.Kerberos = kerberos;
+    }
+}
+
+public struct PostAceBody
+{
+    public Guid Permission;
```

Comment:
This class is maybe missing comments on all it's public elements

Reply:
Ah yes, I was looking to add them in as I fleshed out the Auth and got more familiar with it